homeassistant/components/azure_devops/sensor.py

```diff
@@ -18,76 +19,167 @@
-                    "finish_time": build.finish_time,
-                },
-                build_key=key,
-                organization=entry.data[CONF_ORG],
-                project=project,
-                value=lambda build: build.build_number,
+    [builds, wis] = coordinator.data
+
+    sensors: list[AzureDevOpsSensor] = []
+    sensors.extend(
+        [
+            AzureDevOpsBuildSensor(
+                coordinator,
+                AzureDevOpsBuildSensorEntityDescription(
+                    key=f"{project.id}_{build.definition.id}_latest_build",
+                    name=f"{project.name} {build.definition.name} Latest Build",
+                    icon="mdi:pipe",
+                    attrs=lambda build: {
+                        "definition_id": build.definition.id,
+                        "definition_name": build.definition.name,
+                        "id": build.id,
+                        "reason": build.reason,
+                        "result": build.result,
+                        "source_branch": build.source_branch,
+                        "source_version": build.source_version,
+                        "status": build.status,
+                        "url": build.links.web,
+                        "queue_time": build.queue_time,
+                        "start_time": build.start_time,
+                        "finish_time": build.finish_time,
+                    },
+                    item_key=key,
+                    organization=entry.data[CONF_ORG],
+                    project=project,
+                    value=lambda build: build.build_number,
+                ),
+            )
+            for key, build in enumerate(builds)
+        ]
+    )
+    if wis is not None:
+        sensors.append(
+            AzureDevOpsWorkItemSensor(
+                coordinator,
+                AzureDevOpsWorkItemSensorEntityDescription(
+                    key=f"{project.id}_work_items",
+                    name=f"{project.name} Work Items",
+                    icon="mdi:file-tree",
+                    item_key="",
+                    organization=entry.data[CONF_ORG],
+                    project=project,
+                    value=lambda work_items, _: len(work_items),
+                ),
             ),
         )
-        for key, build in enumerate(coordinator.data)
-    ]
+        sensors.extend(
+            [
+                AzureDevOpsWorkItemSensor(
+                    coordinator,
+                    AzureDevOpsWorkItemSensorEntityDescription(
+                        key=f"{project.id}_type_{work_item_type}_work_items",
+                        name=f"{project.name} {work_item_type} Items",
+                        icon="mdi:file-tree",
+                        item_key=work_item_type,
+                        organization=entry.data[CONF_ORG],
+                        project=project,
+                        value=lambda work_items, work_item_type: len(
```

Comment:
We should not define lambdas that span over multiple lines, create a helper function that you pass instead